src/oc_erchef/apps/chef_db/priv/pgsql_statements.config

```diff
@@ -19,6 +19,14 @@
 
 {ping, <<"SELECT 'pong' as ping LIMIT 1">>}.
 
+{stats, <<"SELECT SUM(seq_scan) as seq_scan, SUM(seq_tup_read) as seq_tup_read, SUM(idx_scan) as idx_scan,"
+          " SUM(idx_tup_fetch) as idx_tup_fetch, SUM(n_tup_ins) as n_tup_ins, SUM(n_tup_upd) as n_tup_upd,"
+          " SUM(n_tup_del) as n_tup_del, SUM(n_live_tup) as n_live_tup, SUM(n_dead_tup) as n_dead_tup,"
+          " SUM(heap_blks_read) as heap_blks_read, SUM(heap_blks_hit) as heap_blks_hit,"
+          " SUM(idx_blks_read) as idx_blks_read, SUM(idx_blks_hit) as idx_blks_hit, SUM(toast_blks_read) as toast_blks_read,"
+          " SUM(toast_blks_hit) as toast_blks_hit, SUM(tidx_blks_read) as tidx_blks_read,"
+          " SUM(tidx_blks_hit) as tidx_blks_hit FROM pg_stat_all_tables, pg_statio_all_tables">>}.
```

Comment:
I think this is inaccurate.  "," does a cartesian join and inflates the number of final rows.  I'm still trying to figure out if there was a join here we wanted or if we just wanted to get a composite query and how to do that (because reading postgresql to me is like spaniard trying to read Brazilian Portuguese...  so close but yet so far).

Reply:
there's no join. just a composite of 2 queries, perhaps i was trying to be too clever